DigitalLearningSolutions.Web/Styles/index.scss

```diff
@@ -29,6 +29,20 @@ a.delete-button:active {
   background-color: #6b140e;
 }
 
+button.delete-button {
+  background-color: #d5281b;
+  box-shadow: 0 4px 0 #6b140e;
+  margin-bottom: 16px !important;

```

Comment:
These styles are copied from the above a.delete-button styles. I'd be against changing the behaviour of them both for now. I suspect the reason it is there is because I've found a lot of the nhs styles use !important, and the only real way to override that is to use more of them